all2vec/__init__.py

```diff
@@ -217,21 +217,28 @@ def save(self, folder):
             json.dump(enttypes, handle)
 
     @classmethod
-    def load(cls, folder):
-        """Load object."""
+    def load(cls, file_getter_or_folder):
+        """Load object.
+
+        file_getter_or_folder -- a FileGetter or a folder string
+        """
+        # to preserve backwards compatibility
+        if isinstance(file_getter_or_folder, str):
```

Comment:
I forget that this lib is compatible with 3.x. Let's ignore the basestring check for now, in that case.